_articles/ib/metrics.md

```diff
@@ -0,0 +1,128 @@
+---
+lang: en
```

Comment:
Hi - these `lang` slugs need to be changed on every page: 
```suggestion
lang: ib
```
And ideally the strings in `description` would also be translated